test/Makefile

```diff
@@ -339,8 +339,12 @@ build-docker-compose: ## Builds docker-compose.yml
 			echo "Error: L2_CHAIN_ID variable is not set. Please provide a value."; \
 			exit 1; \
 	fi
-	@sed -i.bak 's/<l2-chain-id>/$(L2_CHAIN_ID)/g' config/test.node.config.toml
-	@rm -f config/test.node.config.toml.bak
+	@sed -i.bak 's/<l2-chain-id>/$(L2_CHAIN_ID)/g' docker-compose.yml
+	@rm -f docker-compose.yml.bak
+
+.PHONY: deploy-l1-network-contracts
+deploy-l1-network-contracts: ## Deploys contracts to L1 network
+	docker exec -it zkevm-mock-l1-network bash docker/scripts/deploy-contracts.sh
```

Comment:
per my comments in https://github.com/Snapchain/zkevm-contracts/pull/1, I think we just need sth like: https://github.com/Snapchain/zkevm-node/blob/develop/test/docker-compose.yml#L475